.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,150 @@
+name: Issue
+description: Report a bug/issue
+labels: [bug]
+body:
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Confirm the following items before proceeding.
+      options:
+        - label: I've read the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
+          required: true
+        - label: I've searched other issues and no duplicate issues were found.
+          required: true
```

Comment:
@takeshi-iwanari 
I'm sorry, but I mean that this might not necessary also.

```suggestion
```

Reply:
@takam5f2 
Thank you for fixing it. I just forgot to change issue templates.